Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM golang:1.13 AS build-env
+WORKDIR /go/src/github.com/fairwindsops/pluto/
+
+ENV GO111MODULE=on
+ENV GOPROXY=https://proxy.golang.org
+ENV CGO_ENABLED=0
+ENV GOOS=linux
+ENV GOARCH=amd64
+
+COPY go.mod .
```

Comment:
It seems unnecessary and inefficient to copy go.mod, run go mod download, and then copy the entire directory into the container after that in line 14.  It would be best to combine these.

Reply:
This, IIUC, is a best practice when writing Dockerfiles. This way you can change the code without having to rebuild the dependency layer